__fixtures__/test-project/api/src/functions/auth.ts

```diff
@@ -139,6 +139,19 @@ export const handler = async (event, context) => {
       resetTokenExpiresAt: 'resetTokenExpiresAt',
     },
 
+    // Specifies attributes on the cookie that dbAuth sets in order to remember
```

Comment:
Why is this cookie thing included in this PR?

Reply:
Because the fixture wasn't regenerated before